expat/CMakeLists.txt

```diff
@@ -424,11 +425,21 @@ if(EXPAT_BUILD_FUZZERS)
             "execution.")
     endif()
 
+    if(EXPAT_OSSFUZZ_BUILD AND NOT DEFINED ENV{LIB_FUZZING_ENGINE})
+        message(WARNING
```

Comment:
Just an idea: Should we change this to `SEND_ERROR` or is `WARNING` better for something on the outside?

Reply:
I think the former is better because we shouldn't be failing silently as we currently do with a warning.